src/Control/Eff/Cut.hs

```diff
@@ -2,6 +2,8 @@
 {-# LANGUAGE PatternGuards #-}
 {-# LANGUAGE DeriveDataTypeable #-}
 {-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE TypeFamilies #-}
```

Comment:
@qnikst since we already are using CPP in this library, there doesn't seem much drawback of doing that. the benefit is to help keep dependencies separated as they relate to the two open union implementations. i will merge your pull request once that edit is made. if you'd rather i do it, it may take a couple of days as my dev machine is out of commission at the moment

i'm looking for better ways to address this (and am open to suggestion), but that's for future